easybuild/easyconfigs/d/dlb/dlb-3.4-gompi-2023b.eb

```
@@ -0,0 +1,35 @@
+# vim: set syntax=python:
+easyblock = 'ConfigureMake'
+
+name = 'dlb'
+version = '3.4'
+
+description = """
+DLB is a dynamic library designed to speed up HPC hybrid applications (i.e.,
+two levels of parallelism) by improving the load balance of the outer level of
+parallelism (e.g., MPI) by dynamically redistributing the computational
+resources at the inner level of parallelism (e.g., OpenMP). at run time.
+"""
+homepage = 'https://pm.bsc.es/dlb/'
+docurls = ['https://pm.bsc.es/ftp/dlb/doc/user-guide/']
+
+toolchain = {'name': 'gompi', 'version': '2023b'}
+builddependencies = [('Python', '3.11.5', '', ('GCCcore', '13.2.0'))]
```

I don't know what you have tried but this is just the wrong approach. You should be doing this:
```suggestion
builddependencies = [('Python', '3.11.5')]
```